deployer/deployer.py

```diff
@@ -527,8 +540,6 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
             self.set_scheduler(version)
 
     def deploy_elb(self, health_check, health_check_port):
-        self.elbs = [self.app]
-
```

Comment:
Mmm, I think this is a mistake. Nothing fails to me with this change, buy I supose that the elb was not associated with the ASG.

I'll revert this change.